app/src/main/AndroidManifest.xml

```diff
@@ -9,18 +9,16 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-
-        <!--Splash activity launches first, then it starts MainActivity as defined in  SplashActivity.kt-->
-
+        <activity android:name=".LoginActivity"></activity>
+        <activity android:name=".SignupActivity" /> <!-- Splash activity launches first, then it starts MainActivity as defined in  SplashActivity.kt -->
```

Comment:
The comment may be updated, but u can leave it for now, we will do it next time

Reply:
Okay sounds good!